src/main/java/ledgefarm/models/LedgefarmError.java

```diff
@@ -0,0 +1,5 @@
+package ledgefarm.models;
```

Comment:
What is the use of this file as it dont any method/property

Reply:
It was added mistakenly, now removed.